src/programs.c

```diff
@@ -1947,6 +2295,13 @@ static __always_inline int exit_clone(struct pt_regs *ctx)
     ev->u.clone_info = clone_info;
     push_telemetry_event(ctx, ev);
 
+    ev->id = *id;
+    bpf_map_delete_elem(&telemetry_ids, &pid_tgid);
+    ev->done = TRUE;
+    ev->telemetry_type = TE_RETCODE;
+    ev->u.retcode = (u32)PT_REGS_RC(ctx);
```

Comment:
Same here as the other `retcode`.
```suggestion
    ev->u.r.retcode = (u32)PT_REGS_RC(ctx);
```

Reply:
Should be fixed but definitely double check